packages/cadl-ranch-specs/cadl-ranch-summary.md

```diff
@@ -1735,3 +1735,43 @@ Expected input body:
   "for": "value"
 }
 ```
+
+### Unions_sendInt
+
+- Endpoint: `post /unions/int`
+
+This test is testing sending int in simple union property.
+
+```json
+{ "simpleUnion": 1 }
+```
+
+### Unions_sendIntArray
+
+- Endpoint: `post /unions/int-array`
+
+This test is testing sending int array in simple union property.
+
+```json
+{ "simpleUnion": [1, 2] }
+```
+
+### Unions_sendInstallation1
+
+- Endpoint: `post /unions/installation1`
+
+This test is testing sending Installation1 in named union property.
```

Comment:
what is Installation1? We really want the test scenario names to clearly convey what the test is for just through the name. Maybe `sendFirstUnionValue` and `sendSecondUnionValue`?

Reply:
Thanks for the suggestions @iscai-msft , I updated for them.